crates/bevy_math/src/primitives/dim2.rs

```diff
@@ -125,6 +125,90 @@ impl Ellipse {
     }
 }
 
+/// An annulus primitive
```

Comment:
These docs need to be extended / improved: using the exact same words as the type name rarely clarifies things :)

"A primitive shape formed by the region between two circles, also known as a ring."